sleap/io/visuals.py

```diff
@@ -515,6 +536,7 @@ def save_labeled_video(
         fps: Frames per second for output video.
         scale: scale of image (so we can scale point locations to match)
         crop_size_xy: size of crop around instances, or None for full images
+        background: output video background. Either Original, Black, White, Grey
```

Comment:
Match case sensitivity
```suggestion
        background: output video background. Either original, black, white, grey
```